### PR TITLE
Install an RPM without null lines

### PR DESCRIPTION
**Pull request by lucab85**
Install an RPM without null lines

https://bugzilla.redhat.com/show_bug.cgi?id=1691353

### CODE REVIEW

File: dnf-behave-tests/features/microdnf/install.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Feature: Install package
+
+@bz1691353
+Scenario: Install an RPM without null lines
+  Given I use repository "dnf-ci-fedora"
+   When I execute dnf with args "install glibc"
```
Comment (m-blaha) [inline dnf-behave-tests/features/microdnf/install.feature]: Isn't this test supposed to test microdnf?

File: dnf-behave-tests/features/microdnf/install.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Feature: Install package
```
Comment (m-blaha) [inline dnf-behave-tests/features/microdnf/install.feature]: All microdnf tests should be decorated by @no_installroot

File: dnf-behave-tests/features/microdnf/install.feature (hunk 1 above)
Comment (lucab85) [reply dnf-behave-tests/features/microdnf/install.feature]: You're right, I'll adjust the test. It is useful also usage of [PR#731](https://github.com/rpm-software-management/ci-dnf-stack/pull/731)

File: dnf-behave-tests/features/microdnf/install7.feature [hunk 3]
```diff
@@ -0,0 +1,14 @@
+@no_installroot
+Feature: Install package
+
+@bz1691353
+Scenario: Install an RPM without null lines
+  Given I use repository "dnf-ci-fedora"
+   When I execute microdnf with args "install lame"
```
Comment (m-blaha) [inline dnf-behave-tests/features/microdnf/install7.feature]: The test is run on the host - not in the install root. That said, all other repositories already enabled on host (like fedora, updates...) are used which makes the test considerably slower (need to sync all metadata for those repositories) and unable to run without the network (which is the way how gating tests are run).
You can solve this by removing all current repo files as first steps (see also other microdnf/install*.feature files in the master branch):
```
Background:
Given I delete file "/etc/dnf/dnf.conf" 
  And I delete file "/etc/yum.repos.d/*.repo" with globs
```

Comment (lucab85) [reply dnf-behave-tests/features/microdnf/install7.feature]: Thank you for the explanation.

**Pull request merged**